vendor/bat-native-ads/src/bat/ads/ad_content_value_util.cc

```diff
@@ -68,88 +68,103 @@ base::Value::Dict AdContentToValue(const AdContentInfo& ad_content) {
 AdContentInfo AdContentFromValue(const base::Value::Dict& root) {
   AdContentInfo ad_content;
 
-  if (const auto* value = root.FindString(kAdType)) {
-    ad_content.type = AdType(*value);
-  } else if (const auto* value = root.FindString(kLegacyAdType)) {
-    ad_content.type = AdType(*value);
+  if (const auto* ad_type_value = root.FindString(kAdType)) {
+    ad_content.type = AdType(*ad_type_value);
+  } else if (const auto* legacy_ad_type_value =
+                 root.FindString(kLegacyAdType)) {
+    ad_content.type = AdType(*legacy_ad_type_value);
```

Comment:
Are these changes needed?

Reply:
This bump included clang's `-Wshadow` in the overall build process, which disallows shadowed variables. Open to other approaches here, but basically the name does have to change.

Reply:
Nevermind, I see the issue. Will address in a fixup.

Reply:
Fixed in b4174716b278ed73d7aab2c1ed841d54e25cb51a.